org.eclipse.xtend.lib/build.gradle

```diff
@@ -6,3 +6,23 @@ dependencies {
 	compile project(':org.eclipse.xtext.xbase.lib')
 	compile project(':org.eclipse.xtend.lib.macro')
 }
+
+task copyGwtSources(dependsOn: 'build') {
```

Comment:
This [discussion](https://discuss.gradle.org/t/how-do-i-declare-a-dependency-on-a-modules-test-code/7172/9) solves the same problem for test JARs, you can adjust it to the source JAR problem.